src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/OffenceSDSReleaseArrangementLookupService.kt

```diff
@@ -153,11 +153,24 @@ class OffenceSDSReleaseArrangementLookupService(
     val exclusionForOffence = exclusionsForOffences[offenceCode]
 
     return when (exclusionForOffence?.schedulePart) {
-      SDSEarlyReleaseExclusionSchedulePart.SEXUAL -> SDSEarlyReleaseExclusionType.SEXUAL
-      SDSEarlyReleaseExclusionSchedulePart.DOMESTIC_ABUSE -> SDSEarlyReleaseExclusionType.DOMESTIC_ABUSE
-      SDSEarlyReleaseExclusionSchedulePart.NATIONAL_SECURITY -> SDSEarlyReleaseExclusionType.NATIONAL_SECURITY
-      SDSEarlyReleaseExclusionSchedulePart.TERRORISM -> SDSEarlyReleaseExclusionType.TERRORISM
-      SDSEarlyReleaseExclusionSchedulePart.VIOLENT -> if (fourYearsOrMore(sentenceAndOffence)) {
+      SDSEarlyReleaseExclusionSchedulePart.SEXUAL,
+      SDSEarlyReleaseExclusionSchedulePart.SEXUAL_T3,
+      -> SDSEarlyReleaseExclusionType.SEXUAL
+      SDSEarlyReleaseExclusionSchedulePart.DOMESTIC_ABUSE,
+      SDSEarlyReleaseExclusionSchedulePart.DOMESTIC_ABUSE_T3,
+      -> SDSEarlyReleaseExclusionType.DOMESTIC_ABUSE
+      SDSEarlyReleaseExclusionSchedulePart.NATIONAL_SECURITY,
+      SDSEarlyReleaseExclusionSchedulePart.NATIONAL_SECURITY_T3,
+      -> SDSEarlyReleaseExclusionType.NATIONAL_SECURITY
+      SDSEarlyReleaseExclusionSchedulePart.TERRORISM,
+      SDSEarlyReleaseExclusionSchedulePart.TERRORISM_T3,
```

Comment:
Shouldn't these exclusion schedule parts be mapped to the equivalent T3 exclusion types?

Reply:
That's correct. I have these changes on a follow up ticket, but it makes more sense to add them to this branch. I'll make the commit shortly.